### PR TITLE
In 133272507 add suffixand middle name

### PR DESCRIPTION
**Pull request by NateOSI**

### CODE REVIEW

File: app/assets/javascripts/Suffix.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const SUFFIX = Object.freeze({
```
Comment (pemaOSI) [inline app/assets/javascripts/Suffix.js]: Can we call it NAME_SUFFIX?

File: app/assets/javascripts/components/common/ShowField.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  labelClassName: React.PropTypes.string,
+  label: React.PropTypes.string.isRequired,
+  children: React.PropTypes.oneOfType([
+    React.PropTypes.string,
+    React.PropTypes.number,
+  ]),
+}
+export default ShowField
+
+
```
Comment (pemaOSI) [inline app/assets/javascripts/components/common/ShowField.jsx]: two unnecessary new lines?

File: app/assets/javascripts/components/common/ShowField.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const ShowField = ({wrapperClassName, labelClassName, label, children}) => (
```
Comment (pemaOSI) [inline app/assets/javascripts/components/common/ShowField.jsx]: Not sure about ShowField as the name. It seems to have more to do with grid display.
https://getbootstrap.com/examples/grid/
http://getbootstrap.com/css/
From the usage, It looks like we could be creating grid columns. Should we call it GridColumn?

Can we call wrapperClassName to gridClassName? or gridColumnClassName?

Could we rename wrapperClassName

File: app/assets/javascripts/components/common/InputField.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const InputField = ({wrapperClassName, labelClassName, id, label, onChange, value}) => (
```
Comment (pemaOSI) [inline app/assets/javascripts/components/common/InputField.jsx]: if we could call wrapperClassName to gridColumnClassName, it implies we are using grid system.

File: app/assets/javascripts/components/common/DateField.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const DateField = ({wrapperClassName, labelClassName, id, label, onChange, value}) => (
```
Comment (pemaOSI) [inline app/assets/javascripts/components/common/DateField.jsx]: What do you think about merging DateField with InputField component?

Comment (NateOSI) [inline app/assets/javascripts/components/common/DateField.jsx]: We will probably add date related behavior to these fields in the future. Like make them date pickers

File: app/assets/javascripts/components/common/InputField.jsx (hunk 4 above)
Comment (NateOSI) [inline app/assets/javascripts/components/common/InputField.jsx]: done

**Pull request merged**